### PR TITLE
Add `Array::make` macro

### PR DESCRIPTION
**Pull request by BastiaanOlij**
Added support for our `Array::make` macro.

### CODE REVIEW

File: include/godot_cpp/variant/array_helpers.hpp [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+	appendable.append(value);
+	return append_all(appendable, args...);
+}
+
+template <typename T>
+T append_all(T appendable) {
+	return appendable;
+}
+} // namespace helpers
+} // namespace godot
```
Comment (Calinou) [inline include/godot_cpp/variant/array_helpers.hpp:50]: Add a newline at end of file:

```suggestion
} // namespace godot

```

**Review by Calinou — CHANGES_REQUESTED** [review]

File: include/godot_cpp/variant/array_helpers.hpp (hunk 1 above)
Comment (akien-mga) [reply include/godot_cpp/variant/array_helpers.hpp:50]: Still seems missing.

**Comment by BastiaanOlij** [comment]
@akien-mga Sorry, pushed it into godotengine/godot-cpp instead of bastiaanolij/godot-cpp so the fix didn't show up:)

**Review by akien-mga — APPROVED** [review]

**Comment by akien-mga** [comment]
Thanks!

**Pull request merged**